tests/integration/baseTestValidator.py

```diff
@@ -104,7 +104,6 @@ def run_test(self, jobId, statusId, statusName, fileSize, stagingRows,
             stagingRows: Expected number of rows in validation db staging tables. False if no rows are expected
             errorStatus: Expected status in file table of error DB, False if file object should not exist
             numErrors: Expected number of errors
-            rowErrorsPresent: Checks flag for whether row errors occurred, None to skip the check
```

Comment:
Hmm... weird, I was just using PyCharm to tell me when things were missing/extra. I wonder why it didn't tell me that those two needed to be added.